substrate-node/pallets/pallet-tfgrid/src/lib.rs

```diff
@@ -158,7 +158,7 @@ pub mod pallet {
     pub type LocationOf<T> = <T as Config>::Location;
 
     // Input type for serial number
-    pub type SerialNumberInput = BoundedVec<u8, ConstU32<{ node::MAX_SERIAL_NUMBER_LENGTH }>>;
+    pub type SerialNumberInput = Vec<u8>;
```

Comment:
why did you change this? I think having it as 

```
    pub type SerialNumberInput = BoundedVec<u8, ConstU32<{ node::MAX_SERIAL_NUMBER_LENGTH }>>;
```

was correct?

It's used here:

https://github.com/threefoldtech/tfchain/blob/a67df080a6517da9e708908e30f85f3aed4fdd39/substrate-node/pallets/pallet-tfgrid/src/lib.rs#L1026


So this extrinsic will be invalid in the lenghts exceeds the boundedVectors length.